src/test/java/org/springframework/samples/petclinic/rest/VisitRestControllerTests.java

```diff
@@ -127,7 +130,10 @@ public void testGetVisitSuccess() throws Exception {
             .andExpect(status().isOk())
             .andExpect(content().contentType("application/json"))
             .andExpect(jsonPath("$.id").value(2))
-            .andExpect(jsonPath("$.description").value("rabies shot"));
+            .andExpect(jsonPath("$.description").value("rabies shot"))
+            .andExpect(jsonPath("$.adHoc").exists())
+            .andExpect(jsonPath("$.scheduled").exists())
+            .andExpect(jsonPath("$.isPaid").exists());
     }
 
```

Comment:
Missing couple tests : 
+ That we would have error with null vet
+ That we can't to override paid option on create/update
+ That we can't update vet on update
+ Create visit would fail as vet is not set, so need update that test
+ Update test not updated